app/Entirety/smartdatamodels/templates/smartdatamodels/smartdatamodels_list.html

```diff
@@ -0,0 +1,102 @@
+{% extends '_base.html' %}
+{% load static %}
+{% load compress %}
+
+{% block title %}Smart Data Models{% endblock %}
+
+{% block content %}
+    <div class="container">
+    <div class="row">
+            <div class="row height d-flex justify-content-center align-items-center">
+                <div class="col-md-1">
+                    <span class="input-group-btn">
+                        <div class="row">
+                            {% if not view_only %}
+                                <a href="{% url 'projects:smartdatamodels:create' project.uuid  %}" class="btn btn-secondary btn-lg">
+                                    <i class="bi-plus-circle"></i>
+                                </a>
+                            {% endif %}
+                        </div>
+                    </span>
+                </div>
+                <div class="col-md-8">
+                    <form action="">
+                        <div class="p-3 border border-light rounded-pill bg-light">
+                            <div class="input-group col-md-12 border-0 shadow-none">
+                                <input name="search" type="text" class="form-control input-lg border-light rounded-pill"
+                                       placeholder="Search models by name"/>
+                                <span class="input-group-btn">
+                                <button class="btn btn-info btn-lg bg-light border-light" type="submit" value="search">
+                                    <i class="bi-search"></i>
+                                </button>
+                                </span>
+                            </div>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </div>
+
+
+    <div class="d-flex flex-row flex-wrap">
+        <div class="row row-cols-1 row-cols-md-auto g-4 mt-3 mb-3">
+            {% for model in smartdatamodel_list %}
+                <div class="col mt-5">
+                    <div class="card h-100">
+                        <div class="card-header d-flex justify-content-center align-items-center">
+                            <h5>{{ model.name }}
+                                <i class="bi bi-filetype-json"></i>
+                            </h5>
+                        </div>
+
+                        <div class="card-body">
+                             <p class="d-inline-block text-secondary">Created by: {{ model.owner.first_name|add:' '|add:model.owner.last_name }}</p>
+                            <br>
+                             <p class="d-inline-block text-secondary">Last modified by: {{ model.last_modified_by.first_name|add:' '|add:model.last_modified_by.last_name }}</p>
+                        </div>
+                            <div class="ms-3 mb-3">
+                                <a href="{% url 'projects:smartdatamodels:update' project.uuid model.uuid %}"
```

Comment:
@djs0109 If the user role with "view_only" can not edit but only view the smart model then can you please change the title from Edit to Inspect?

Reply:
@djs0109 the button classes for Inspect and Delete are different, can you please use the same button as from Delete?
![image](https://github.com/N5GEH/n5geh.tools.entirety/assets/79838286/cdfe6068-409d-4dd4-94fb-f5c614d6b630)